entities/markets/market.ts

```diff
@@ -40,7 +40,7 @@ export class Market {
   }
 
   isLoser(index: number) {
-    return this.answer && this.answer !== index;
+    return this.answer !== null && this.answer !== index;
```

Comment:
Maybe this also works? Or is there a reason to check if the value is explicitly null?
```suggestion
    return this?.answer !== index;
```

Reply:
we need to check for null values, othis the same as the above and was not working, cause if it's null it will make this statement true.